examples/manipulation-demo.launch.py

```diff
@@ -45,32 +92,25 @@ def generate_launch_description():
                     ]
                 ),
             ),
-            IncludeLaunchDescription(
-                PythonLaunchDescriptionSource(
-                    [
-                        "src/examples/rai-manipulation-demo/Project/Examples/panda_moveit_config_demo.launch.py",
-                    ]
-                ),
-            ),
-            # Launch the robotic_manipulation node after 3 seconds, to ensure the moveit node is ready
-            RosTimer(
-                period=3.0,
-                actions=[
-                    Node(
-                        package="robotic_manipulation",
-                        executable="robotic_manipulation",
-                        name="robotic_manipulation_node",
-                        output="screen",
-                    ),
-                ],
+            # Launch the game launcher and wait for it to load
+            launch_game_launcher,
+            RegisterEventHandler(
+                event_handler=OnProcessStart(
+                    target_action=launch_game_launcher,
+                    on_start=[
+                        wait_for_game_launcher,
+                    ],
+                )
             ),
-            # Launch the game launcher
-            ExecuteProcess(
-                cmd=[
-                    LaunchConfiguration("game_launcher"),
-                    "-bg_ConnectToAssetProcessor=0",
-                ],
-                output="screen",
+            # Launch the MoveIt node after loading the simulation
+            RegisterEventHandler(
+                event_handler=OnExecutionComplete(
+                    target_action=wait_for_game_launcher,
+                    on_completion=[
+                        launch_moveit,
+                        launch_robotic_manipulation,
```

Comment:
I would also not start open-set vision nodes if game launcher is not starting

Reply:
I agree, this is fixed now.